src/pages/methodology/index.tsx

```diff
@@ -11,7 +10,6 @@ const Methodology = () => {
 
   return (
     <Layout title={"Methodology"}>
```

Comment:
Looks like we have removed the CPR methodology page?

Reply:
Correct - it was never actually navigable and so I thought best to remove it from the repo. The current methodology is hosted on github here: https://github.com/climatepolicyradar/methodology